app/helpers/layout_helper.rb

```diff
@@ -95,8 +95,13 @@ def stylesheet(*args)
     content_for(:stylesheets) { stylesheet_link_tag(*args) }
   end
 
-  def javascript(*args)
-    content_for(:javascripts) { javascript_include_tag(*args) }
+  def javascript(*args, overrides)
+    # Workaround for overriding javascript load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if overrides.is_a?(Hash) && overrides[:source] == "webpack_asset_paths"
+      content_for(:javascripts) { overrides[:webpacked] }
+    else
+      content_for(:javascripts) { javascript_include_tag(*args) }
+    end
   end
```

Comment:
Looking at `javascript_include_tag` docs (https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-javascript_include_tag) it seems that we should support **kwargs as well since they can be used as options to pass into `javascript_include_tag`.

I'd suggest a bit different solution:

```suggestion
  def javascript(*args, **kwargs)
    # Workaround for overriding javascript load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
    deprecated_source = args.first
    if deprecated_source.is_a?(Hash) && deprecated_source[:source] == "webpack_asset_paths"
      content_for(:javascripts) { deprecated_source[:webpacked] }
    else
      content_for(:javascripts) { javascript_include_tag(*args, **kwargs) }
    end
  end
```

Reply:
`deprecated_source`  didnt have deprecated_source[:webpacked] or source, changed to kargs.
Looks like the openscap issue only happens when a user clicks that button before the page is fully loaded, once its loaded it works fine